src/com/company/model/DeleteProvider.java

```diff
@@ -0,0 +1,24 @@
+package com.company.model;
+
+import com.company.view.ConsoleReader;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by yulia on 23.02.17.
+ */
+public class DeleteProvider {
```

Comment:
This class is using to get list of `DeleteColumnDefiniton`. Maybe name should be like **DeleteColumnDefinitionProvider** or something similar